build.gradle

```diff
@@ -250,7 +251,20 @@ test {
 
 changelog {
     header.set("[${propertyString('mod_version')}] - ${LocalDate.now().format("yyyy-MM-dd")}")
-    headerParserRegex.set(~/(\d+\.\d+)/)
+    if (propertyString('mod_version').split("\\.").length == 2) headerParserRegex.set(~/(\d+\.\d+)/)
```

Comment:
KeepAChangelog adheres to SemVar, would this be fine?

Reply:
This is only a temporary patch, the correct one should create regex directly 🤭